pyproject.toml

```diff
@@ -12,11 +12,12 @@ authors = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = ">=3.8,<3.12"
```

Comment:
It would be good to update the GitHub workflow to add 3.11 to make sure this works. (in `.github/workflows/test.yml`)